ChaosMod/Effects/db/Misc/MiscNothing.cpp

```diff
@@ -1,6 +1,40 @@
 #include <stdafx.h>
+static std::string options[20] =
+{ "Nothing",
+"All Peds Are Peds",
+"Teleport To Current Location",
+"Expanded & Enhanced",
+"Spawn Air",
+"Destroy All Destroyed Vehicles",
+"Kill All Dead Peds",
+"+0 Wanted Stars",
+"Jesus Watches Over You",
+"Set Time To Current Time",
+"Set Player Into Current Vehicle",
+"All Cops Are Cops",
+"Aim to Point Gun",
+"Give Everyone A Nose",
+"Teleport Player A Few Millimeters",
+"Flying Birds",
+"Teleport All Elephants To Player",
+"PC Experience",
+"All Enemies Attack Player",
+" "};
 
-static RegisterEffect registerEffect(EFFECT_NOTHING, nullptr, nullptr, nullptr, EffectInfo
+
+static void OnStart()
+{
+
+	int start = GET_GAME_TIMER();
+	std::string override = options[g_Random.GetRandomInt(0, options->size() - 1)];
+	while (start + 25000 > GET_GAME_TIMER())
```

Comment:
25s might be too long, maybe like. 5 - 10?

Reply:
Why this loop? Wouldn’t it be „funnier“ if it just appears and  confuses and not be obviously „the nothing effect, that always does this“?

Reply:
Take a look at how it's being used in the Fake Teleport OnStart -> https://github.com/gta-chaos-mod/ChaosModV/blob/master/ChaosMod/Effects/db/Player/PlayerTpController.cpp#L281

You should just be able to set it to what you want, and then wait and it'll change back after the delay. No need for a loop. 

```cpp
g_pEffectDispatcher->OverrideEffectName(...);
WAIT(25000);
```